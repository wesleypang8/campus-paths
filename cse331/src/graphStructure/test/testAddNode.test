#test adding node to graph

#Create graph
CreateGraph graph1

#Add a node
AddNode graph1 node1

#print nodes
ListNodes graph1