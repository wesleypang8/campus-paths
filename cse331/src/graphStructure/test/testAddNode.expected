#test adding node to graph

#Create graph
created graph graph1

#Add a node
added node node1 to graph1

#print nodes
graph1 contains: node1